src/utils/src/Math.c

```diff
@@ -20,21 +20,22 @@ STATUS computePower(UINT64 base, UINT64 exponent, PUINT64 result) {
             (base <= 1000 && exponent <= 6), STATUS_INVALID_ARG);
 
     if (base == 0) {
-        return 0;
+        *result = 0;
```

Comment:
Perhaps this should return `1` in the case of `0^0` (which is mathematically undefined) but all implementations I could find return either 1 or the equivalent of NaN.